cinema/serializers.py

```diff
@@ -68,13 +85,52 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionListSerializer(many=False, read_only=True)
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+
+
+class TicketListSerializer(TicketSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat")
```

Comment:
The `TicketListSerializer` inherits from `TicketSerializer`, but it redefines the `Meta` class without adding any new fields or overriding existing ones. If the intention is to limit the fields, ensure that the `fields` attribute in `TicketListSerializer` is correctly set to include only the desired fields.